src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.Shared.xml

```diff
@@ -62,4 +62,13 @@
       <method signature="System.Void Initialize()" />
     </type>
   </assembly>
+
+  <!-- methods used by hot reload.
+       TODO: once there's a feature flag, add it to the linker descriptor
```

Comment:
Can we use the `Debugger.IsSupported` feature switch for now? That's what we did for `MetadataUpdateHandlerAttribute`.

https://github.com/dotnet/runtime/blob/636988a0727d01479f9af94ef1c56bb7e79ecb24/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.Shared.xml#L33-L36

I can put a PR to fix this.

Reply:
Yes, that makes sense to me.

Reply:
I opened https://github.com/dotnet/runtime/pull/51994